dtfs-central-api/src/helpers/get-fee-record-payment-entity-group-reconciliation-data.test.ts

```diff
@@ -0,0 +1,187 @@
+import { difference } from 'lodash';
+import { when } from 'jest-when';
+import { FEE_RECORD_STATUS, FeeRecordEntityMockBuilder, UtilisationReportEntity, UtilisationReportEntityMockBuilder } from '@ukef/dtfs2-common';
+import { getFeeRecordPaymentEntityGroupReconciliationData } from './get-fee-record-payment-entity-group-reconciliation-data';
+import { FeeRecordPaymentEntityGroup } from '../types/fee-record-payment-entity-group';
+import { TfmUsersRepo } from '../repositories/tfm-users-repo';
+import { aTfmUser } from '../../test-helpers';
+
+describe('getFeeRecordPaymentEntityGroupReconciliationData', () => {
```

Comment:
can we use the same format as other files please: get-fee-record-payment.....

Reply:
This does match the same format as the other helper test files